lib/spack/spack/test/conftest.py

```diff
@@ -66,12 +67,18 @@ def ensure_configuration_fixture_run_before(request):
         request.getfixturevalue("mutable_config")
 
 
+@pytest.fixture(scope="session")
+@pytest.mark.skipif(not spack.util.git.get_git(), reason="requires git to be installed")
```

Comment:
Looking [at the docs](https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-mark-skip-ref), it seems `skip` and `skipif` are always applied to test functions, not to fixtures. Does the `skipif` above propagate to tests using the fixture?

Reply:
yeah good catch -- apparently you have to use `pytest.skip()`